pages/common/zip.md

```diff
@@ -29,3 +29,7 @@
 - Delete files from an existing zip file:
 
 `zip -d {{compressed.zip}} "{{foo/*.tmp}}"`
+
+- Archive a directory and its contents to a multi-part [s]plit zip file (i.e. 3GB parts):
```

Comment:
I assume other zip sizes can be specified?

It may be better to use `e.g.` instead of `i.e.` 👍 `i.e.` implies that it may have to be 3GB, whereas `e.g.` would be saying that 3GB is an example and can be replaced by other sizes.

Reply:
yes, you can specify other sizes. thank you for the correction. 👍